### DISCUSSION TITLE
Increment and decrement action for hotkeys

### DISCUSSION THREAD

**Discussion by fifolifo** [xanderfrangos/twinkle-tray#874] [Ideas]
I really appreciate the ability to create hotkeys to control volume. I was expecting the ability to configure hotkeys to raise or lower the volume each time I entered the hotkey, but this doesn't appear to be possible with the currently available actions. 

It would be great if there was an action that would increment or decrement by the value entered. For now, I'm using the "Cycle list of values" action with five different volume levels that meets the majority of my needs and this works well for me right now.  Thanks for developing this very helpful app.

**Comment by matek11523** [comment]
Hi,

You can use Adjust Value action for this. Just setup two hotkeys, in one put +# and -# in the other. Pressing the hotkeys then will adjust a volume respectively.
